src/kirby/components/list/list.component.ts

```diff
@@ -75,13 +75,29 @@ export class ListComponent implements OnInit, OnChanges {
    * - list, if {@link #isSectionsEnabled} is `false`
    * - section, if {@link #isSectionsEnabled} is `true`
    *
-   * `square` means **without** rounded corners, `rounded` means **with** rounded corners.
+   * `square` means **without** rounded corners, `rounded` means **with** rounded corners.,  `none` means **without** padding, border, box-shadow and background.
    */
   @Input() shape: ListShape = 'rounded';
   @HostBinding('class.rounded')
   public get isRounded(): boolean {
     return this.shape === 'rounded';
   }
+  @HostBinding('class.no-shape')
+  public get isNoShape(): boolean {
+    return this.shape === 'none';
+  }
+
+  /**
+   * Determine spacing between list items:
+   * - list, if {@link #isSectionsEnabled} is `false`
+   *
+   * `true` means **with** space between list items, `false` means **without** any space between list items.
+   */
+  @Input() itemSpacing: boolean = false;
```

Comment:
Consider naming: `hasItemSpacing` if type is boolean
OR
keep `itemSpacing` but use a [string literal type](https://www.typescriptlang.org/docs/handbook/advanced-types.html#string-literal-types):
```suggestion
  @Input() itemSpacing: 'none' | 'separate' = 'none';
```
(We could also "steal" from flexbox, even though it would probably just be confusing: `space-between`)

Reply:
I have moved it to an attribute selector. Guess we should call it: has-item-spacing="true"